### DISCUSSION TITLE
Unable to create profile (resolved)

### DISCUSSION THREAD

**Discussion by TMRE** [audiamus/BookLibConnect#143] [General]
I am no longer able to create a profile. When I login to Audible, it takes me to the solve the puzzle and shows a page not found.

![1](https://github.com/audiamus/BookLibConnect/assets/78712812/cb3ed5eb-35bc-4804-afad-7ff71f0dfd35)
![2](https://github.com/audiamus/BookLibConnect/assets/78712812/a30bf160-ff55-4333-8ecd-05eb0994865c)

**Comment by TMRE** [comment]
Issue resolved with clearing browser history.

**Reply by TomPaine56** [reply]
somehow i have the same problem now on Mac as well as Windows, with Opera or Chrome Browser, clearing browser history does not resolve the issue in these cases, the problem lies with the Germany selection.

**Comment by TMRE** [comment]
I restarted the server and it fixed the issue

**Comment by Alandrias** [comment]
Same problem.. cant create profil.
Account-Datas not accept, but correct.
after 20 times i have pass die Account-Data-Problem and now....
After giving chapta the next thing i see is:
Suchst du etwas?
Tut uns leid: Die Web-Adresse, die du eingegeben hast, gibt es auf unserer Website nicht.
 Zur Amazon.de [Homepage](https://www.amazon.de/)

**Reply by audiamus** [reply]
Basically, you are in. No fault at all. Both the BookLibConnect profile dialog and the manual state this outcome (in English) as the expected result. Copy the final URL from your Web browser into the respective field in the dialog.
